manifests/init.pp

```diff
@@ -113,6 +119,30 @@
   Boolean $epel_gpg_managed                            = true,
   $os_maj_release                                      = $epel::params::os_maj_release,
 ) inherits epel::params {
+  $_epel_metalink = $epel::params::epel_metalink_enabled ? {
+    true    => $epel::params::epel_metalink,
+    default => undef,
+  }
+  $_epel_testing_metalink = $epel::params::epel_testing_metalink_enabled ? {
+    true    => $epel::params::epel_testing_metalink,
+    default => undef,
+  }
+  $_epel_source_metalink = $epel::params::epel_source_metalink_enabled ? {
+    true    => $epel::params::epel_source_metalink,
+    default => undef,
+  }
+  $_epel_debuginfo_metalink = $epel::params::epel_debuginfo_metalink_enabled ? {
+    true    => $epel::params::epel_debuginfo_metalink,
+    default => undef,
+  }
+  $_epel_testing_source_metalink = $epel::params::epel_testing_source_metalink_enabled ? {
+    true    => $epel::params::epel_testing_source_metalink,
+    default => undef,
+  }
+  $_epel_testing_debuginfo_metalink = $epel::params::epel_testing_debuginfo_metalink_enabled ? {
+    true    => $epel::params::epel_testing_debuginfo_metalink,
```

Comment:
I think these should all be local variables. The unit test failures for this are caused by it.

Reply:
fixed, thanks